jwst/associations/lib/rules_level3.py

```diff
@@ -565,7 +565,7 @@ def __init__(self, *args, **kwargs):
 
 
 @RegistryMarker.rule
-class Asn_Lv3SpectralSource(AsnMixin_Spectrum):
+class Asn_Lv3NRCWFSS(AsnMixin_Spectrum):
     """Level 3 slit-like, multi-object spectrographic Association
```

Comment:
These seem like pretty generic descriptions for a rule that specifically applies to only NIRCam WFSS mode. "Level 3 slit-like, multi-object" sounds to me like it should also include NIRSpec MOS, but it doesn't.

Reply:
Also, what's the purpose of the references to `nrs_autoflat` and `nrs_autowave` in the rules below? What do these have to do with NIRCam WFSS?

Reply:
good catch. those should be removed.